recipes/rfc5424-logging-handler/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "rfc5424-logging-handler" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ae14073ef6d76d0c730ad6b6e3aeece841a6d413672d282876c0506dc097257
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+    - pytz
+    - setuptools
+    - tzlocal
+  run:
+    - python
+    - pytz
+    - tzlocal
+
+test:
+  imports:
+    - rfc5424logging
+
+about:
+  home: https://github.com/jobec/rfc5424-logging-handler
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
The license file is not being packaged. Could you please raise this issue on this project?
For now you can add it manually

Reply:
How does one do that? `curl` in the LICENSE file from the repo?

Reply:
Fixed it by specifying it as an additional source